README.md

```diff
@@ -1,22 +1,32 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+  # Capstone Module 1: Hemp Industries Submmit 2021
 
-> One paragraph statement about the project.
+> This is the final project regading the firt module of Microverse program. 
+> The main subjects that we intenselly worked on, were the three pilars of any 
+> website on mobile or desktop devices, wich are HTML,CSS & JavaScript. I have to confess that, 
+> even when I spare lots of hours reading the basics and some advanced features, It was 
+> one of the biggest intellectual challengues I've been trough in all my live. I will enhance
+> you to keep reading this document and find what I personally exprienced over the past 5 weeks
+> of intense learning, lots of sleep depravation, frustrations and satisfactions over this period 
+> of time. 
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+As I mentioned before, this is the final project of the module. We are applying all the skills we accordion-body adquired week over week on a variety of tasks and applied all of them in a project
+with the major challenge to make it all in just one week. Languages,Frameworks,Technologies, and 
+lots of so called professional or soft skills. It was awesome to work with people from all around
+the world.
 
 ## Built With
 
-- Major languages: HTML 5, CSS 3, JavaScript ES6, GIT.
-- Frameworks: BootStrap 4.
-- Technologies used: VScode, GITHUB, Canva. 
+- Major languages: HTML, CSS, JavaScript, GIT.
+- Frameworks: Bootstrap 5, TypScript, NPM.
+- Technologies used: GitHub, VScode, OpenAI, Linterns.
```

Comment:
- [ ] Kindly in this section you should only list tools that you use in this project and remove the rest.

Reply:
Done.